src/commons/services/access/providers/ifbs-access-provider.js

```
@@ -44,33 +44,111 @@ class IfbsAccessProvider extends AccessProvider {
     };
   }
 
+  async getStatus(accessPoint, bookingContext) {
+    const client = await this._getClient(bookingContext.tenant);
+    const usage = this._resolveUsageWindowStatus(bookingContext);
+
+    if (bookingContext.lastOpenBoxId) {
+      try {
+        const result = await client.monitorOpenBox(bookingContext.lastOpenBoxId);
```

ESLint

[error] Replace `bookingContext.lastOpenBoxId` with `⏎··········bookingContext.lastOpenBoxId,⏎········`

(prettier/prettier)